runtime/src/main/java/org/corfudb/runtime/view/SealChainReplication.java

```diff
@@ -0,0 +1,53 @@
+package org.corfudb.runtime.view;
+
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.clients.BaseClient;
+import org.corfudb.runtime.exceptions.QuorumUnreachableException;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * Seals the Log unit servers in the Chain Replication View
+ *
+ * Created by zlokhandwala on 3/7/17.
+ */
+@Slf4j
+public class SealChainReplication implements ISealReplicationView {
+
+    /**
+     * Seals the Layout Segment if atl east one log unit server is sealed in every stripe.
+     * Else throws exception
+     *
+     * @param runtime       Runtime to set remote epoch for log unit servers.
+     * @param layoutSegment Layout segment to be sealed.
+     * @param sealEpoch     Epoch number to seal the servers with
+     * @return  True if seal successful
+     * @throws QuorumUnreachableException   Thrown if seal unsuccessful
+     */
+    @Override
+    public Boolean sealLayoutSegment(CorfuRuntime runtime, Layout.LayoutSegment layoutSegment, long sealEpoch) throws QuorumUnreachableException {
```

Comment:
For Chain replication there I do not see big benefit in sealing only one log unit. Datapath will still be blocked till every log unit in the chain is upgraded to the new epoch.